sdk/cosmos/azure-cosmos/azure/cosmos/_default_retry_policy.py

```diff
@@ -54,11 +54,12 @@ def __init__(self, *args):
 
     def needsRetry(self, error_code):
         if error_code in DefaultRetryPolicy.CONNECTION_ERROR_CODES:
-            if len(self.args) > 0:
+            if self.args:
                 if (self.args[4]["method"] == "GET") or (http_constants.HttpHeaders.IsQuery in self.args[4]["headers"]):
                     return True
                 return False
             return True
+        return False
```

Comment:
Technically speaking this function returned `None` at this point before, but I believe `False` matches the intent